Phonebook.Frontend/src/styles.scss

```diff
@@ -52,8 +52,22 @@ app-root {
 .square-img50 {
   width: 50px;
   height: 50px;
+  display: block;
 }
 
+.square-img40 {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+.square-img36 {
+  width: 36px;
+  height: 36px;
+  display: block;
+}
```

Comment:
Do you want to add a mixin for this classes? 
You can find an example in `mixins.scss` used like `@include mat-icon-size(60px);`
Something like this...
```
@mixin square-img-size($size) {
  width: $size;
  height: $size;
  display: block;
}
```
Is `display: block` necessary?

Reply:
Thanks for the hint, the display is nacessary for making the image round, without it the image would be a square